plenum/server/catchup/cons_proof_service.py

```diff
@@ -14,6 +15,10 @@
 
 logger = getlogger()
 
+ConsProofReady = NamedTuple('LedgerCatchupStart',
```

Comment:
I would move it into `node_messages.py`. The same reasons as in the previous PR :)